docs(ee-license) Clarify and update license and Bintray doc (#2376)
Kong/docs.konghq.com: app/enterprise/2.1.x/deployment/access-license.md

Kong Enterprise requires a license file. This guide walks you through how to access your license file.

<div class="alert alert-ee blue">
<strong>Note:</strong> The following guide only pertains to paid versions of
Kong Enterprise. For free trial information, see our list of ways to
<a href="/enterprise/{{page.kong_version}}/introduction/#try-kong-enterprise">
try Kong Enterprise</a>, or see your free trial sign-up email.
</div>

## Prerequisites

Before you can access your license, you need to sign up for a Kong Enterprise
subscription. With this subscription, you will receive access to the Kong
Bintray repository, which contains the necessary license files.

If you have purchased a subscription but haven't received a license file,
contact your sales representative.

## Accessing your license file through a browser

1. Log into [https://bintray.com/](https://bintray.com/login?forwardedFrom=%2Fkong%2F).
If you are unaware of your login credentials, reach out to your CSE and they'll
be able to assist you.

> 2. You will notice that along with Kong Enterprise, there is a new

Reviewer: ```suggestion
2. Notice that along with Kong Enterprise, there is a new
```

repository that has the same name as your company. Click on that repo.

3. In the repo, click on the file called **license**.

    ![bintray-license](/assets/images/docs/ee/access-bintray-license.png)

4. Click into the **Files** section

    ![bintray-license-files](/assets/images/docs/ee/access-bintray-license-files.png)

> 5. Click any file you would like to download.

Reviewer: ```suggestion
5. Click any file you want to download.
```

## Programmatically accessing your license file

For programmatic access you'll need 3 pieces of information:

>  - Username (was provided by email by your CSE)

Reviewer: ```suggestion
 - Username (was provided through email by your CSE)
```
